validations/src/mainnet_validations.rs

```diff
@@ -12,8 +12,9 @@ const FIRST_EMERGENCY_COMMITTEE: [&str; 7] = [
     "wit1etherz02v4fvqty6jhdawefd0pl33qtevy7s4z",
 ];
 
-// FIXME(#1791): decide first hard fork epoch
-const FIRST_HARD_FORK: u32 = u32::MAX;
+/// 22 January 2021 @ 09:00:00 UTC
+const FIRST_HARD_FORK: u32 = 192000;
```

Comment:
```suggestion
const FIRST_HARD_FORK: Epoch = 192000;
```
It is exactly the same, but could be more descriptive reading the code